spec/services/problem_check/sidekiq_check_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.describe ProblemCheck::SidekiqCheck do
+  subject(:check) { described_class.new }
+
+  describe ".call" do
+    context "when Sidekiq processed a job recently" do
+      before do
+        Jobs.stubs(:last_job_performed_at).returns(1.minute.ago)
+        Jobs.stubs(:queued).returns(1)
+      end
+
+      it { expect(check).to be_chill_about_it }
+    end
+
+    context "when last job processed was a long time ago, but no jobs are queued" do
+      before do
+        Jobs.stubs(:last_job_performed_at).returns(7.days.ago)
+        Jobs.stubs(:queued).returns(0)
+      end
+
+      it { expect(check).to be_chill_about_it }
+    end
+
+    context "when no jobs have ever been processed, but no jobs are queued" do
+      before do
+        Jobs.stubs(:last_job_performed_at).returns(nil)
+        Jobs.stubs(:queued).returns(0)
+      end
+
+      it { expect(check).to be_chill_about_it }
+    end
+
+    context "when no jobs were processed recently and some jobs are queued" do
+      before do
+        Jobs.stubs(:last_job_performed_at).returns(20.minutes.ago)
+        Jobs.stubs(:queued).returns(1)
+      end
+
+      it do
+        expect(check).to have_a_problem.with_priority("low").with_message(
+          'Sidekiq is not running. Many tasks, like sending emails, are executed asynchronously by Sidekiq. Please ensure at least one Sidekiq process is running. <a href="https://github.com/mperham/sidekiq" target="_blank">Learn about Sidekiq here</a>.',
```

Comment:
This is contentious...I prefer to check the actual `I18n.t("blah", {})` text in tests rather than reproducing it in tests...but also not a hill I would die on

Reply:
I'm okay with either. I think there might be a risk that using the actual `I18n` will mask "translation missing for ..." problems, but I think there's a setting we can enable in the test environment to raise an error when a translation is missing.

It would be nice to not have to fiddle with these longs strings everywhere.

Reply:
Yeah I've never been overly concerned with the masking of "translation missing". I think it's more preferable to have better devex by not putting the strings everywhere. A missing translation is a very minor issue IMO, and definitely should have an automated check in test environments :+1:

Reply:
Will do this in a separate PR. 👍